src/templates/professional/components/Education.tsx

```diff
@@ -11,11 +13,7 @@ export const Education = ({ education }: { education: EducationIntrf[] }) => {
             </p>
             <div className="flex justify-between items-center">
               <p className="font-normal text-[#1AB0B3]">{item.institution}</p>
-              <div className="flex gap-3">
-                <p className="text-xs">
-                  {item.startDate} - {item.endDate}
-                </p>
-              </div>
+              <p className="gap-3 text-xs">{`${item.startDate} - ${item.endDate}`}</p>
```

Comment:
This file has been updated. Can you pull from v2 branch and update this PR?

Reply:
Thanks. This is done.